phing/build.yml

```diff
@@ -72,11 +72,18 @@ git:
     pre-commit: ${blt.root}/scripts/git-hooks
     commit-msg: ${blt.root}/scripts/git-hooks
 
+# Site(s) definition.
 multisite:
-  # The docroot/sites/default directory is used by default.
-  name:
-    - default
-    # - another_site
+  default:
+    aliases:
```

Comment:
I suppose the `drush.aliases` could go away entirely and be replaced with a `sites.default.yml` (or `sites/default.yml`) config file that would be in the template by default and would be loaded if no other config is specified; if this is something you're not adversed to.